models/resource.js

```diff
@@ -30,6 +30,7 @@ module.exports = (sequelize, DataTypes) => {
       email: DataTypes.STRING,
       website: DataTypes.STRING,
       address: DataTypes.TEXT,
+      descriptivenote: DataTypes.TEXT,
```

Comment:
Let's make this variable name just `description` or use lower-camelCase `descriptiveNote`...